### ISSUE TITLE
mask did not use?

### ISSUE THREAD

**Issue by SYSUykLin** [OpenGVLab/DragGAN#111]
I note that the drag_gan function at draggan.py have a parameter "mask", and it did not use.

And 
`        # if mask is not None:
        #     loss += ((F - F0) * (1 - mask)).abs().mean() * lam`
has been mask“#”
